tracer/src/Datadog.Trace/HttpHeaderNames.cs

```diff
@@ -37,8 +37,15 @@ public static class HttpHeaderNames
         public const string Origin = "x-datadog-origin";
 
         /// <summary>
-        /// Origin of the distributed trace.
+        /// The user agent that originated an http request.
```

Comment:
Unrelated to this PR, but this comment was copied from above and it not correct.